### PR TITLE
Added manual power level, speed fan, airkare, relax and standby mode

### PR DESCRIPTION
**Pull request by shisva**

### CODE REVIEW

File: edilkamin/api.py [hunk 1]
```diff
@@ -201,3 +201,148 @@ def set_target_temperature(token: str, mac_address: str, temperature: int) -> st
     return mqtt_command(
         token, mac_address, {"name": "enviroment_1_temperature", "value": temperature}
     )
+
+
+def device_info_get_fan_1_speed(info: typing.Dict) -> int:
+    """Get fans speed value from cached info."""
+    return info["status"]["fans"]["fan_1_speed"]
```
Comment (AndreMiras) [inline edilkamin/api.py:208]: I'm thinking we could generalize most of these functions using `fan: int`:
```python
def device_info_get_fan_speed(info: typing.Dict, fan: int) -> int:
```

File: edilkamin/api.py [hunk 2]
```diff
@@ -201,3 +201,148 @@ def set_target_temperature(token: str, mac_address: str, temperature: int) -> st
...
+    """Get fans speed value."""
+    info = device_info(token, mac_address)
+    return device_info_get_fan_2_speed(info)
+
+
+def set_fan_2_speed(token: str, mac_address: str, fan_2_speed: int) -> str:
+    info = device_info(token, mac_address)
+    fan_number_confiuguration = info["nvm"]["installer_parameters"]["fans_number"]
+    
+    if fan_number_confiuguration >= 2:
```
Comment (AndreMiras) [inline edilkamin/api.py:240]: We probably want to return a warning otherwise

File: edilkamin/api.py [hunk 3]
```diff
@@ -201,3 +201,148 @@ def set_target_temperature(token: str, mac_address: str, temperature: int) -> st
...
+        """
+        Set fan 2 speed.
+        Return response string e.g. "Command 0123456789abcdef executed successfully".
+        """
```
Comment (AndreMiras) [inline edilkamin/api.py:244]: The docstring should go after the function definition

File: edilkamin/api.py [hunk 4]
```diff
@@ -201,3 +201,148 @@ def set_target_temperature(token: str, mac_address: str, temperature: int) -> st
...
+        return mqtt_command(token, mac_address, {"name": "fan_3_speed", "value": fan_3_speed})
+
+
+def device_info_get_airkare(info: typing.Dict) -> int:
+    """Get airkare status from cached info."""
+    return info["status"]["commands"]["airkare_function"]
+
+
+def get_airkare(token: str, mac_address: str) -> int:
```
Comment (AndreMiras) [inline edilkamin/api.py:276]: I think that function actually returns a boolean rather than an integer

File: edilkamin/api.py [hunk 5]
```diff
@@ -201,3 +201,148 @@ def set_target_temperature(token: str, mac_address: str, temperature: int) -> st
...
+
+def set_airkare(token: str, mac_address: str, airkare: bool) -> str:
+    """
+    Set airkare.
+    Return response string e.g. "Command 0123456789abcdef executed successfully".
+    """
+    return mqtt_command(token, mac_address, {"name": "airkare_function", "value": airkare})
+
+
+def device_info_get_relax_mode(info: typing.Dict) -> int:
```
Comment (AndreMiras) [inline edilkamin/api.py:290]: Return type is probably a bool here too

File: edilkamin/api.py [hunk 6]
```diff
@@ -201,3 +201,148 @@ def set_target_temperature(token: str, mac_address: str, temperature: int) -> st
...
+
+def set_manual_power_level(token: str, mac_address: str, manual_power_level: int) -> str:
+    """
+    Set manual power level value.
+    Return response string e.g. "Command 0123456789abcdef executed successfully".
+    """
+    return mqtt_command(token, mac_address, {"name": "power_level", "value": manual_power_level}) 
+
+
+def device_info_get_standby_mode(info: typing.Dict) -> int:
```
Comment (AndreMiras) [inline edilkamin/api.py:328]:
```suggestion
def device_info_get_standby_mode(info: typing.Dict) -> bool:
```

File: edilkamin/api.py [hunk 7]
```diff
@@ -201,3 +201,148 @@ def set_target_temperature(token: str, mac_address: str, temperature: int) -> st
...
+    """Get standby mode status."""
+    info = device_info(token, mac_address)
+    return device_info_get_standby_mode(info)
+
+
+def set_standby_mode(token: str, mac_address: str, standby_mode: bool) -> str:
+    info = device_info(token, mac_address)
+    is_auto = info["nvm"]["user_parameters"]["is_auto"]
+    
+    if is_auto == True:
```
Comment (AndreMiras) [inline edilkamin/api.py:343]: https://peps.python.org/pep-0008/#programming-recommendations
> Don’t compare boolean values to True or False using ==:
```suggestion
    if is_auto:
```

**Review by AndreMiras — APPROVED** [review]
Awesome, thanks for the feature push.
I wrote a few comments, feel free to address otherwise I will try to address them myself later.
I also try to keep the test coverage high, but I can pick that one up

File: edilkamin/api.py (hunk 3 above)
Comment (shisva) [reply edilkamin/api.py:244]: Ok done.

File: edilkamin/api.py (hunk 5 above)
Comment (shisva) [reply edilkamin/api.py:290]: Ok done.

File: edilkamin/api.py (hunk 4 above)
Comment (shisva) [reply edilkamin/api.py:276]: Ok done.

**Review by AndreMiras — APPROVED** [review]
Thanks for addressing, it looks good enough, I'll merge and address the rest myself

**Pull request merged**